src/components/Atoms/Link/Link.js

```diff
@@ -5,7 +5,7 @@ import StyledLink from './Link.style';
 
 const Link = ({ children, color, href, target, type, home, ...rest }) => {
   const window = target === 'blank' ? '_blank' : '_self';
-  const relationship = target === 'blank' ? 'noopener noreferrer' : false;
+  const relationship = target === 'blank' ? 'noopener noreferrer' : 'false';
```

Comment:
Let's remove `rel` altogether. `rel="false"` is not correct. I think the only place using `rel` is our `logo` link. Logo should add this attribute.

Reply:
Ok sure